test/wikidata_ids_decorator_test.rb

```diff
@@ -1,21 +1,51 @@
 require 'test_helper'
 
 describe WikidataIdsDecorator do
-  it 'has a version number' do
-    ::WikidataIdsDecorator::VERSION.wont_be_nil
+  describe 'gem' do
+    it 'has a version number' do
+      ::WikidataIdsDecorator::VERSION.wont_be_nil
+    end
   end
+end
+
+describe WikidataIdsDecorator::Links do
+  around { |test| VCR.use_cassette(File.basename(url), &test) }
+  let(:response) { Scraped::Request.new(url: url).response }
+  let(:decorator) { WikidataIdsDecorator::Links.new(response: response) }
 
   describe 'Wikipedia page with links' do
+    let(:url) { 'https://en.wikipedia.org/wiki/Douglas_Adams_(disambiguation)' }
+
     it 'adds a data-wikidata attribute to links' do
-      body = <<-HTML
-<div id="bodyContent">
-  <a href="https://en.wikipedia.org/wiki/Douglas_Adams" title="Douglas Adams">Douglas Adams</a>
-</div>
-      HTML
-      response = Scraped::Response.new(body: body, url: 'https://en.wikipedia.org/wiki/Douglas_Adams_(disambiguation)')
-      decorator = WikidataIdsDecorator::Links.new(response: response)
-      decorator.body.must_include '<a href="https://en.wikipedia.org/wiki/Douglas_Adams" title="Douglas Adams"' \
-        ' wikidata="Q42">Douglas Adams</a>'
+      decorator.body.must_include '<a href="/wiki/Douglas_Adams" title="Douglas Adams" wikidata="Q42">'
+    end
+  end
+
+  describe 'Anchored links' do
+    let(:url) { 'https://en.wikipedia.org/wiki/Life,_the_Universe_and_Everything' }
+
+    it 'adds a data-wikidata attribute to anchored links' do
+      decorator.body.must_include '<a href="/wiki/OCLC#Identifiers_and_linked_data" title="OCLC" wikidata="Q190593">'
+    end
+
+    it 'does not add a data-wikidata attribute to anchored links pointing to a place within the same page' do
+      decorator.body.must_include '<a href="#Plot_summary">'
+    end
+  end
+
+  describe 'Redirect links' do
+    let(:url) { 'https://en.wikipedia.org/wiki/Life,_the_Universe_and_Everything' }
+
+    it 'adds a data-wikidata attribute to redirect <links>  </links>' do
```

Comment:
I don't really follow what the `<links>  </links>` is doing here. I'm also not sure that this really explains well what's being tested — surely this test is about making sure that it links to the _correct_ Wikidata item — i.e. after following the redirect?